source/ast/expressions/AssertionExpr.cpp

```diff
@@ -1125,6 +1452,27 @@ void ConditionalAssertionExpr::serializeTo(ASTSerializer& serializer) const {
         serializer.write("else", *elseExpr);
 }
 
+std::optional<SequenceRange> CaseAssertionExpr::computeSequenceLengthImpl() const {
+    std::optional<SequenceRange> max;
+    for (const auto& item : items) {
+        if (const auto itemLen = item.body->computeSequenceLength(); itemLen.has_value()) {
+            if (const auto itemLenVal = itemLen.value(); max.has_value() || max < itemLenVal)
```

Comment:
Is this meant to be `!max.has_value()` ?

Reply:
yes it's typo - fixed